File: Node-Project/daily-project/typeorm/src/entity/box.ts
import {Entity, PrimaryGeneratedColumn, Column} from "typeorm";

@Entity('boxes')
export class Box {

    @PrimaryGeneratedColumn()
    id: number;

    @Column({name:'code'})
    code: string;

    @Column({name:'company_id'})
    companyId: string;

    @Column({name:'name'})
    name: string;

}

File: springboot/src/main/java/com/shuoquan/springboot/entity/Box.java
package com.shuoquan.springboot.entity;


import com.baomidou.mybatisplus.annotation.TableName;
import java.io.Serializable;
import lombok.Data;
import lombok.experimental.Accessors;

@TableName("boxes")
@Data
@Accessors(chain = true)
public class Box implements Serializable {

  private long id;
  private String code;
  private long companyId;
  private long groupId;
  private String name;
  private String address;
  private long lastUseAt;
  private long managerEndAt;
  private long status;
  private long createdAt;
  private long threshold;
  private String qrcodeUrl;
  private String stock;
  private long areaId;
  private String deviceSerial;

}
